config/serverless.es.yml

```diff
@@ -63,6 +63,7 @@ data_visualizer.resultLinks.fileBeat.enabled: false
 
 # Search Playground
 xpack.searchPlayground.ui.enabled: true
+xpack.searchPlayground.ui.queryBuilder.enabled: false
```

Comment:
Since we only need this feature flag while developing what would you think about using a ui setting instead? You can see an example of this in the [Search Homepage setup PR](https://github.com/elastic/kibana/pull/186224). Since we don't want to show the setting in the UI you don't have to register the setting you just set the value and read it `uiSettings.get<boolean>(FEATURE_FLAG_ID, false);`

Reply:
That was what I was going for initially, would that be visible in settings menu? If no I can switch to that

Reply:
Also gives us an advantage to write FTRs and triggering the UI setting before the test is run.

Reply:
suggestion: could we call this UI setting `searchMode` instead of query builder? May cause confusion because query builder refers to the query editor thats accessible in both search and chat.